Tasks/UseRubyVersionV0/Tests/L0FindVersionInLinuxCache.ts

```diff
@@ -0,0 +1,41 @@
+import ma = require('vsts-task-lib/mock-answer');
+import tmrm = require('vsts-task-lib/mock-run');
+import path = require('path');
+import fs = require('fs');
+import os = require('os');
+import tool = require('vsts-task-tool-lib/tool');
+
+let taskPath = path.join(__dirname, '..', 'main.js');
+let tr: tmrm.TaskMockRunner = new tmrm.TaskMockRunner(taskPath);
+
+tr.setInput('versionSpec', '2.5');
+tr.setInput('addToPath', 'false');
+
+process.env['AGENT_VERSION'] = '2.116.0';
+
+// provide answers for task mock
+let a: ma.TaskLibAnswers = <ma.TaskLibAnswers>{
+    
+};
+tr.setAnswers(a);
+
+tr.registerMock('vsts-task-tool-lib/tool', {
+    findLocalTool: () => path.join('/', 'Ruby', '2.5.4')
+});
+
+fs.symlinkSync = (s) => {
+}
+fs.unlinkSync = (s) => {
+}
+fs.existsSync = (s) => {
+    return true;
+}
+tr.registerMock('fs', fs);
```

Comment:
I feel like rebinding actual functions on the `fs` module could cause a confusing bug.  Also, if you use a brand-new mock object like you did with tool-lib above, it will tell you right away that you forgot to mock something, where using the actual module will sneak through silently.